byon/files/seaclouds-catalog.bom

```diff
@@ -186,13 +186,14 @@ brooklyn.catalog:
       type: org.apache.brooklyn.entity.brooklynnode.BrooklynNode
       id: deployer
       name: Deployer
-      version: 0.8.0-incubating
+      version: 0.9.0-SNAPSHOT
```

Comment:
I think it is too risky. Let's close this PR, `seaclouds-catalog.bom` has to be deeply modified.

Focus on having a good documentation in the README.md, where you exactly specify what's needed to have the SeaClouds Deployer (Apache Brooklyn + ? ), how to install it, how to run it, etc. See https://github.com/SeaCloudsEU/SeaCloudsPlatform/issues/164